### PR TITLE
update scala example nueral-style 

### PR DESCRIPTION
**Pull request by Ldpe2G**
follow changes in  #1964

### CODE REVIEW

**Comment by Ldpe2G** [comment]
all test on osx were failed,


Files changed in this PR includes, display the incomplete path
**********************************
ain/scala/ml/dmlc/mxnet/Symbol.scala
scala-package/examples/src/main/scala/ml/dmlc/mxnet/examples/neuralstyle/ModelVg
scala-package/examples/src/main/scala/ml/dmlc/mxnet/examples/neuralstyle/NeuralS
(END) 

 anyone knows the reason?

**Comment by javelinjs** [comment]
Seems to be something wrong with the CI. Let's wait some time and I'll trigger another build.

**Comment by Ldpe2G** [comment]
ok, thanks.

File: scala-package/examples/src/main/scala/ml/dmlc/mxnet/examples/neuralstyle/NeuralStyle.scala [hunk 1]
```diff
@@ -153,45 +190,41 @@ object NeuralStyle {
...
       val optimState = optimizer.createState(0, img)
 
       logger.info(s"start training arguments $alle")
 
-      var oldImg = img.copy()
-      var gradArray: Array[NDArray] = null
+      var oldImg = img.copyTo(dev)
+      val clipNorm = img.shape.toVector.reduce(_ * _)
+      val tvGradExecutor = getTvGradExecutor(img, dev, alle.tvWeight)
+//      var gradArray: Array[NDArray] = null
```
Comment (javelinjs) [inline scala-package/examples/src/main/scala/ml/dmlc/mxnet/examples/neuralstyle/NeuralStyle.scala]: remove

**Comment by javelinjs** [comment]
Please merge the master.

**Comment by Ldpe2G** [comment]
ok

**Comment by javelinjs** [comment]
It is really weird that all tests hang on osx.
@piiswrong It seems that you met this problem in your recent PR as well, how did you solve it?

**Comment by piiswrong** [comment]
Just restart it

**Comment by Ldpe2G** [comment]
still failed on osx

**Comment by javelinjs** [comment]
@Ldpe2G How about open another pr and let's see whether it can work around.

**Comment by Ldpe2G** [comment]
ok, I will try

**Pull request closed without merging**